getting-started/kubearmor_visibility.md

```diff
@@ -0,0 +1,128 @@
+# KubeArmor Visibility
+
+KubeArmor currently supports enabling visibility for containers/hosts.
+
+Visibility for containers is enabled by default and can be accessed using the `karmor` tool.
+
+
+<details>
+  <summary>Know More On Visibillity</summary>
+
+#### KubeArmor provides visibility on the following behavior of containers
+* Process
+* Files
+* Networks
+
+</details>
+
+
+## Getting Container Visibility
+
+##### Prerequisites
+
+* If you don't have access to a K8s cluster, please follow  [this](https://github.com/kubearmor/KubeArmor/blob/main/getting-started/deployment_guide.md#prerequisites) to set one up.
+* To install karmor CLI tool in the cluster follow [here](https://github.com/kubearmor/KubeArmor/blob/main/getting-started/deployment_guide.md#1-download-and-install-karmor-cli-tool)
```

Comment:
> * To install karmor CLI tool in the cluster follow [here](https://github.com/kubearmor/KubeArmor/blob/main/getting-started/deployment_guide.md#1-download-and-install-karmor-cli-tool)

* karmor CLI tool: [Download and install karmor-cli](https://github.com/kubearmor/KubeArmor/blob/main/getting-started/deployment_guide.md#1-download-and-install-karmor-cli-tool)